install_config/install/host_preparation.adoc

```diff
@@ -319,6 +319,33 @@ devices, which is not supported for production use and only appropriate for
 proof of concept environments. For production environments, you must create a
 thin pool logical volume and re-configure Docker to use that volume.
 
+Docker stores images and containers in a graph driver (a pluggable storage technology), such as *DeviceMapper*, 
+*OverlayFS*, and *Btrfs*. Each has advantages and disadvantages. For example, OverlayFS is faster than DeviceMapper 
+at starting and stopping containers, but is not Portable Operating System Interface for Unix (POSIX) compliant 
+because of the architectural limitations of a union file system and is not supported prior to Red Hat Enterprise
+Linux 7.2. See the Red Hat Enterprise Linux release notes for information on using OverlayFS with your version of RHEL.
```

Comment:
"See the Red Hat Enterprise Linux release notes" should be a link to the release notes

Reply:
@ahardin-rh Which version of the RHEL release notes should we link to? The links below are different, one to 7.3 and one to 7.4.  The 7.3 and 7.4 RN are slightly different in their treatment of OverlayFS.